examples/react-main-thread-function/src/App.css

```
@@ -0,0 +1,112 @@
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 8px;
+}
+
+.Logo--react {
+  width: 100px;
+  height: 100px;
+}
+
+@keyframes Logo--spin {
```

Stylelint (17.3.0)

[error] Expected keyframe name "Logo--spin" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)